index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>Portfolio Sneak Peak</title>
+</head>
+<body>
+  <div class="container">
+    <header>
+      <img class="status-bar" src="images/status-bar.png" alt="Portfolio Status Bar">
```

Comment:
- [ ] The hamburger button is on the right, and has a `nav` tag. You are currently missing the hamburger button.
- [ ] [OPTIONAL] I suggest using a [Font Awesome ](https://fontawesome.com/v4.7/icons/)icon instead of SVGs.